packages/taiko-client/driver/chain_syncer/event/blocks_inserter/pacaya.go

```diff
@@ -154,7 +175,7 @@ func (i *BlocksInserterPacaya) InsertBlocks(
 				parent,
 			)
 			if err != nil {
-				log.Debug("Failed to check if batch is in canonical chain already", "batchID", meta.GetBatchID(), "err", err)
+				log.Warn("Failed to check if batch is in canonical chain already", "batchID", meta.GetBatchID(), "err", err)
```

Comment:
Shall we also update the wording in this `WARN`, since most of time its not `Failed to`, but just because batch is not preconfed.